assets/js/app/notifications/index.js

```diff
@@ -2,4 +2,19 @@ import $ from 'jquery';
 
 $(document).ready(function() {
   $('.toast').toast('show');
+  // add a js class to indicate we have JS enabled. Needs a change to either modernizr or something custom @bobdenotter
```

Comment:
Why is this code in `notifications`? I think there ought to be a better place for this.

Reply:
I deleted the styling for the narrow stuff because I thought we were going to ditch it for V4. But I understand differently now. I''ll put it back in place, but it needs work together with the HTML. For the location of the JS, it's temporary, so we have to discuss where to put this and other generic UI js.

Reply:
This has been transferred to common.js